### PR TITLE
Restore exit handler

### PR DESCRIPTION
**Pull request by gravitypriest**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1597308

### CODE REVIEW

File: insights/client/insights_spec.py [hunk 1]
```diff
@@ -44,7 +44,8 @@ def get_output(self):
         '''
         # all commands should timeout after a long interval so the client does not hang
         # prepend native nix 'timeout' implementation
-        timeout_command = 'timeout %s %s' % (self.config.cmd_timeout, self.command)
+        timeout_command = 'timeout -s KILL %s %s' % (
```
Comment (kylape) [inline insights/client/insights_spec.py]: Did you consider using `-k` instead of starting off with the `KILL`?

Comment (gravitypriest) [reply insights/client/insights_spec.py]: I did but I thought that might affect our timeout interval.

**Comment by gravitypriest** [comment]
Changing the command timeout seems prudent. Currently if even one command takes that long, the process already hits our systemd/cron timeout.

File: insights/client/insights_spec.py (hunk 1 above)
Comment (kylape) [reply insights/client/insights_spec.py]: imho if we lover the command timeout, that shouldn't be a big concern.  we could even just do `-k 5` to give a small buffer for graceful termination.

**Comment by kylape** [comment]
What do you think for a default command timeout?  60 seconds?

**Comment by gravitypriest** [comment]
I'm in favor of `-k`. As for timeout, I'm not really sure what would be appropriate. Evidently there are some machines out there taking a long-ass time with some commands. Maybe 2 mins to be safe?

Makes me think we should really start including the elapsed time in the upload headers again.

**Comment by kylape** [comment]
Sure 2m works for me.  It would be great if we included elapsed time in the logs.

**Pull request merged**